use/profiles.js

```diff
@@ -218,173 +211,166 @@ export const useProfiles = () => {
+      tips: t('profile.styles.tips'),
     },
     {
       name: 'gender',
-      key: 'profile.gender',
-      component: 'TInputButtons',
+      label: t('profile.gender.label'),
+      type: 'buttons',
       hideSearchBox: true,
       poster: true,
       options: genderList,
     },
     {
       name: 'type',
-      key: 'profile.type',
-      component: 'TRadioGroup',
+      label: t('profile.type'),
+      type: 'radio',
       hideSearchBox: true,
       poster: true,
       options: typeList,
     },
     {
       name: 'username',
-      key: 'profile.username',
+      label: t('profile.username.label'),
       register: true,
       poster: true,
       required: true,
-      component: 'TInputUsername',
-      before: 'Use only letters, numbers, underscores and periods.',
+      type: 'username',
+      before: t('profile.username.before'),
     },
     {
       name: 'place',
-      label: 'Your city',
+      label: t('profile.place.label'),
       register: true,
       poster: true,
       required: true,
-      component: 'TInputPlace',
-      placeholder: 'City',
+      type: 'place',
+      placeholder: t('profile.place.placeholder'),
     },
     {
       name: 'hometown',
-      key: 'profile.hometown',
+      label: t('profile.hometown.label'),
       required: true,
-      component: 'TInputPlace',
-      before: 'Where were you born and raised?',
+      type: 'place',
+      before: t('profile.hometown.before'),
     },
     {
       name: 'visibility',
-      key: 'profile.visibility',
+      label: t('profile.visibility.label'),
       poster: true,
       component: 'TInputButtons',
       options: visibilityOptions,
-      description:
-        '- Public profiles are searchable in Google and used in our social media to attract new members.\n- Members profile are only visible for logged-in users.\n- Unlisted profiles are possible to open with exact link, but they are not listed in members lists and search.',
+      description: t('profile.visibility.description'),
     },
     {
       name: 'permission',
-      label: 'Do you want free promo?',
+      label: t('profile.permission.label'),
       poster: true,
       component: 'TInputButtons',
       options: booleanOptions,
-      description:
-        'Send us link to your profile on [Instagram](https://instagram.com/wedancevip) and we will promote it on our social media channels: Telegram, Instagram, Facebook and Twitter.',
+      description: t('profile.permission.description'),
     },
     {
       name: 'name',
-      key: 'profile.name',
+      label: t('profile.name.label'),
       required: true,
-      placeholder: '(Required)',
-      before: 'If you want to remain anonym use your first name.',
+      placeholder: t('profile.name.placeholder'),
+      before: t('profile.name.before'),
     },
     {
       name: 'locales',
-      key: 'profile.languages',
-      component: 'TInputLanguages',
+      label: t('profile.locales'),
+      type: 'languages',
```

Comment:
Since #118 you should use `component` with component name instead of mapped type, that we used before.
In the whole file here, not only this field.

Please make sure you add only your changes and don't touch anything else when rebasing or merging.